src/alchemlyb/parsing/amber.py

```diff
@@ -65,8 +65,10 @@ def __init__(self, filename):
         self.filename = filename
         try:
             self.fileh = anyopen(self.filename, 'r')
-        except Exception as ex:  # pragma: no cover
-            logger.exception("Cannot open file %s", filename)
+        except Exception as ex:
+            msg = f"Cannot open file {filename}"
+            logger.exception(msg)
```

Comment:
Keep the original logger line as we don't need to repeat the message in the exception.
```suggestion
            logger.exception("Cannot open file %s", filename)
```

Reply:
Done!